funcx_sdk/funcx/tests/unit/test_atomiccontroller.py

```diff
@@ -0,0 +1,84 @@
+import random
+
+from funcx.sdk.executor import AtomicController
+
+
+class _SS:
+    _start: bool = False
+    _stop: bool = False
+
+    def reset(self):
+        self._start = self._stop = False
+
+    def start(self):
+        self._start = True
+
+    def stop(self):
+        self._stop = True
+
+    @property
+    def started(self) -> bool:
+        return self._start
+
+    @property
+    def stopped(self) -> bool:
+        return self._stop
+
+
+_ss = _SS()
```

Comment:
Maybe instead of having a global object with `reset()` it would make more sense to construct a new object for each test?

i.e. Switch to
```
def __init__(self):
    self._start = False
    self._stop = False
```

and each test replaces `_ss.reset()` with `x = DummyControllerTarget()`?

In fact, since it's just for tests, I'd go with
```python
class DummyTarget:
    def __init__(self) -> None:
        self.started = False
        self.stopped = False

    def start(self) -> None:
        self.started = True

    def stop(self) -> None:
        self.started = True
```

I'm not sure that more than that is needed?

Reply:
Cool; started life as "StartStop" which seemed useless so went with SS.  It's simple enough I didn't worry about it.  I'll make those changes.

Reply:
Refactored.